winterfell/webapp/src/js/modalChangePassword.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+var app = angular.module('vetafiApp');
+app.directive("modalChangePassword", ["net", function(net) {
+    return {
+        restrict: "A",
+        link: function(scope, elem, attrs) {
+            function displayError(msg) {
+              var errorMsg = $(elem).find('.error-msg');
+              errorMsg.text(msg);
+            }
+
+            scope.savePassword = function() {
+              var oldPwd = $(elem).find('input.old').val();
```

Comment:
You don't need to use jQuery to find the value of this input, if you use `ng-model` it will be bound to the `scope` argument which is passed to this directive.